CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## 0.8.0
+
+### Breaking changes
+
+- Ruff 0.8 ([#14383](https://github.com/MichaReiser/ruff/pull/14383))
+
+### Preview features
+
+- Mark UP043 fix unsafe when the type annotation contains any comments ([#14458](https://github.com/MichaReiser/ruff/pull/14458))
```

Comment:
Huh, why did it use my fork as remote

Reply:
probably https://github.com/zanieb/rooster/issues/42?